pkg/operator/latestrevisionclient.go

```diff
@@ -22,18 +24,20 @@ type OpenshiftDeploymentLatestRevisionClient struct {
 var _ revisioncontroller.LatestRevisionClient = OpenshiftDeploymentLatestRevisionClient{}
 
 func (c OpenshiftDeploymentLatestRevisionClient) GetLatestRevisionState() (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, int32, string, error) {
-	o, err := c.TypedClient.OpenShiftAPIServers().Get("cluster", metav1.GetOptions{})
+	ctx := context.TODO() // needs support in library-go
```

Comment:
this requires support in `library-go`, what's our current plan, is somebody working on it ?

Reply:
the context exists in starter.go, it has to be passed down.... not urgent, but I haven't seen a plan...

this is post-rebase todo I guess

/cc @sttts @marun